src/bundle/Resources/public/js/alloyeditor/src/plugins/ez-embed.js

```diff
@@ -179,12 +178,14 @@
                  * @method handleContentLoaded
                  * @param {Object} hits The result of content search
                  */
-                handleContentLoaded: function (hits) {
+                handleContentLoaded: function(hits) {
                     const isEmbedImage = this.element.$.classList.contains(IMAGE_TYPE_CLASS);
                     const content = hits.View.Result.searchHits.searchHit[0].value.Content;
 
                     if (isEmbedImage) {
-                        const fieldImage = content.CurrentVersion.Version.Fields.field.find(field => field.fieldTypeIdentifier === 'ezimage');
+                        const fieldImage = content.CurrentVersion.Version.Fields.field.find(
```

Comment:
Maybe it should be extracted to a method that looks for an `ezimage` fieldtype? I see it can be reused in many places.